src/Symfony/Component/Runtime/Runner/Symfony/HttpKernelRunner.php

```diff
@@ -24,13 +25,24 @@ class HttpKernelRunner implements RunnerInterface
     public function __construct(
         private readonly HttpKernelInterface $kernel,
         private readonly Request $request,
+        private readonly bool $debug = false,
     ) {
     }
 
     public function run(): int
     {
         $response = $this->kernel->handle($this->request);
-        $response->send();
```

Comment:
What if a custom response class customizes its send method ? The method is not final.

Reply:
Yes, that's a problem for sure. Should we make it final then? Or we could try to add an optional $finishRequest arg to it?

Reply:
Good idea the extra argument!